### PR TITLE
feat: text card added

### PR DESCRIPTION
**Pull request by mayuresh-tech**
Work done:
Text cards added (Responsive)

Bugs:
Sidebar:
If the user tests responsive then the sidebar elements get invisible.
aside needs to be given static height (for now set to 500vh)

### CODE REVIEW

File: documentation/docs.html [hunk 1]
```diff
@@ -793,6 +827,50 @@ <h5>You need to wrap element that need to be overlay and on which it is to be ov
...
+                        style="width: 100%; max-height: auto; border:0; transform: scale(1); overflow:hidden;"
```
Comment (kushalsourav) [inline documentation/docs.html:870]: avoid using inline styles

**Review by kushalsourav — APPROVED** [review]
good to merge

File: documentation/docs.html (hunk 1 above)
Comment (mayuresh-tech) [reply documentation/docs.html:870]: noted

Comment (mayuresh-tech) [reply documentation/docs.html:870]: Will resolve in development branch

**Review by manish-GH — APPROVED** [review]
Except for the sidebar issue, everything looks clean

File: components/card/card.css [hunk 2]
```diff
@@ -122,6 +122,30 @@
     rgba(0, 0, 0, 0.3) 0px 1px 3px -1px;
 }
 
+.card-container-text-horizontal {
+  margin: 1rem 1rem;
+  padding: 0.5rem;
+  border: #e6e6e6 2px solid;
```
Comment (deekshithmd) [inline components/card/card.css:128]: try to give color variable

**Review by deekshithmd — APPROVED** [review]

**Pull request merged**